.github/workflows/pytest.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       # Check for missing messages in .po files
       - name: Check for missing messages in .po files
         run: |
-          ./makemessages.sh --all
+          ./makemessages.sh
```

Comment:
Let's keep `--all` to make it explicit within the workflow, that we want "all" languages, not the current default of the script.

Reply:
*meeehhhh* OK. ;)